exasol_integration_test_docker_environment/cli/commands/health.py

```diff
@@ -0,0 +1,51 @@
+import sys
+from inspect import cleandoc
+
+from exasol_integration_test_docker_environment.cli.cli import cli
+from exasol_integration_test_docker_environment.doctor import (
+    health_checkup,
+    recommend_mitigation,
+)
+
+
+@cli.command()
+def health():
```

Comment:
We need to add this command to the readme

Reply:
done